base/src/main/java/com/smartdevicelink/managers/lifecycle/BaseLifecycleManager.java

```diff
@@ -405,6 +411,10 @@ public void onReceived(RPCMessage message) {
                             //If the vehicle is acceptable and this is the first check, init security lib
                             setSecurityLibraryIfAvailable(vehicleType);
                         }
+                        // HAX: Issue #1690, Ford Sync bug returning incorrect display capabilities (https://github.com/smartdevicelink/sdl_java_suite/issues/1690). Store the initial capabilities if we are a media app so that we can use them in the future.
+                        if (appConfig.appType.contains(AppHMIType.MEDIA)) {
+                            initialMediaCapabilities = raiResponse.getDisplayCapabilities();
```

Comment:
is RegisterAppInterfaceResponse. getDisplayCapabilities deprecated because we don't want it to be public anymore?